_layouts/documentation.html

```diff
@@ -24,4 +24,17 @@ <h1 class="documentation-title">{{ page.title }}</h1>
     <div class="documentation-content">{{ content }}</div>
   </article>
 </div>
+<div id='discourse-comments'></div>
+
+<script type="text/javascript">
+ record_from_txt = document.getElementsByClassName('documentation-source')[0].getElementsByTagName("a")[0].href
+  DiscourseEmbed = { discourseUrl: 'http://meta.discourse.org/',
+                     topicId: parseInt(txt.substring(txt.lastIndexOf("/")+1)) };
```

Comment:
It failing should only call out to a missing topic; returning no content. I've instead added a check to make sure the topicID is coming from a meta.discourse.org URL regardless (but it should be looked over anyway, I haven't tried it in a deployed version yet.)